services/graphql-server/src/graphql/definitions/platform/content/index.js

```diff
@@ -382,18 +386,42 @@ input ContentPublishingMutationInput {
   unpublished: Date
 }
 
+input ContentNameMutationInput {
+  "The content ID"
+  id: Int!
+  "The name of the content"
+  value: String
+  "The mutation to target"
```

Comment:
Update this to say `The mutation to target. The default/non-mutated value will be used when this is null or unset.`